composer.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "require": {
     "php": ">=5.3.3",
-    "orkestra/transactor": ">=1.0.0,<1.3.x-dev"
+    "orkestra/transactor": "dev-update/MajorVersion-Symfony3"
```

Comment:
So I think this should now point at `1.2.x-dev`. Should we cut `1.2.0` for `orkestra-transactor`? Then this project can require `>=1.2`.

I think either way, this bundle will need a major version bump